sql/updates/post-update.sql

```diff
@@ -30,3 +30,48 @@ $$;
 
 -- can only be dropped after views have been rebuilt
 DROP FUNCTION IF EXISTS _timescaledb_internal.cagg_watermark(oid);
+
+-- For objects that are newly created, we need to set the initprivs to
+-- the initprivs for the "chunk" catalog table since that is created
+-- in the first version of TimescaleDB and should have the correct
```

Comment:
Could you rewrite the comment a bit so that it is clear 1) we choose chunk table (but it could be any other catalog table
2) The comment about first version can be misinterpreted as version 1.0, and I did. Instead reword it as, **-- the first time Timescaledb extension was installed --**

Reply:
Updated the comment